### ISSUE TITLE
Icon Request: Modrinth

### ISSUE THREAD

**Issue by StartsMercury** [simple-icons/simple-icons#7488]
### Brand Name

Modrinth

### Website

https://modrinth.com

### Popularity Metric

![image](https://user-images.githubusercontent.com/89975834/172771894-c44e232a-5a7c-4596-9bc7-e2ea3c5c77d6.png)

**Global Rank:** 82,584

### Official Resources for Icon and Color

svg: https://github.com/modrinth/knossos/blob/master/assets/images/logo.svg
color: (Found in https://github.com/modrinth/knossos/blob/master/assets/styles/global.scss)
 - dark-mode: [`#1bd96a` <sup>source</sup>](https://github.com/modrinth/knossos/blob/43b8941f7a0f5718384d1c31547703b97e39ab72/assets/styles/global.scss#L106)
 - light-mode: [`#30b27b` <sup>source</sup>](https://github.com/modrinth/knossos/blob/43b8941f7a0f5718384d1c31547703b97e39ab72/assets/styles/global.scss#L22)

### Additional Comments

 - **Are there multiple options for the logo and/or color?**
   
   Yes, as far as I know, there are two. One for both the dark theme and the light theme.
   
 - **Is the icon released under a license?**
   
   All Rights Reserved - https://github.com/modrinth/knossos/blob/master/COPYING.md#logo-licenses
   
 - **If you think the brand might not be accepted, why do you think it should be considered?**
   
   Modrinth is new compared to another Minecraft modding platform Curseforge. I don't want to compare the two more than is necessary, but their website and how things are arranged (in my opinion) is nice (It also have dark theme). 
  
   Scrolling down in their website https://modrinth.com, will feature more what Modrinth has to offer.

---

> [Modrinth - Official Feed The Beast Wiki](https://ftb.fandom.com/wiki/Modrinth)
> Modrinth is a free and open-source modding platform currently in beta.

<sub>Various pages shown with dark mode theme, in 33% zoom. The homepage (left), sodium mod page (upper-right), and fabulously optimized mod pack page (bottom-right).</sub>
![image](https://user-images.githubusercontent.com/89975834/172777438-7a3692e1-b26c-4ffa-af47-927a61d1a447.png)

### Contributing

No

**Comment by StartsMercury** [comment]
> ### Contributing
> No

I'm honestly not confident to be the one working on this icon.

**Comment by Andre601** [comment]
Having the modrinth logo on SimpleIcons would be really cool and I tried to make a matching SVG, but they use a really weird SVG where it's like a empty square and some strokes for the actual lines.

I tried everything I could to convert it into an SVG SimpleIcons would accept, but there's just too much random stuff.

If anyone else wants to try it, the SVG can be obtained from https://github.com/modrinth/art

**Comment by magicus** [comment]
This was attempted twice in PRs that have been closed. The verdict of these PRs was that the "favicon" version of the logo, which is simplified and easier to understand in small icons etc, should be preferred over the full logo.

The "favicon" logo is (despite the name) available from official sources as an svg: https://github.com/modrinth/art/blob/main/Branding/Favicon/favicon.svg

However, it was apparently not fulfilling all technical requirements of simplicity that simple-icons has set up for inclusion, and converting it to that presented quite some work. I have not worked much with svgs so I did not fully understand the problems, but I intend to take up the challenge. Wish me luck! :-)
